### ISSUE TITLE
Flexibilizar a coluna `directory_column` para permitir qualquer coluna ser mapeada a qualquer outra coluna do banco de dados.

### ISSUE THREAD

**Issue by rdahis** [basedosdados/website#205]
Nomes (?): original_column, back_column. @d116626

Exemplo de uso: indicadores econômicos puxando colunas de tabelas originais para VIEWs.

- [ ] Fazer o site ter dropdowns pra conjuntos, tabelas, colunas.
Gerar endpoint que retorna isso.

A pensar:

* Como guardar esses metadados quando colunas são operações mais complexas de outras (várias) colunas?
Guardar direto a query?

**Comment by laura-l-amaral** [comment]
@rdahis  não sei se consegui visualizar bem o exemplo. 
A ideia seria facilitar para as pessoas conectarem informações de diferentes tabelas, num esquema de click and drop?

**Comment by rdahis** [comment]
Não... A ideia é conseguir manter um metadado de linhagem de dados, para colunas que sejam geradas a partir de outras. Exemplo é `pib per capita = pib / população`, que viria de duas colunas diferentes. Eu gostaria de ter todo o mapeamento de relações entre tabelas, colunas, pra ver como está tudo conectado. Mas não vejo como prioridade agora.
